package.json

```diff
@@ -22,8 +19,8 @@
     "@babel/register": "^7.12.1",
     "@babel/runtime": "^7.12.5",
     "@open-web3/dev-config": "^0.1.13",
-    "@polkadot/api": "^4.16.2",
-    "@polkadot/typegen": "^4.16.2",
+    "@polkadot/api": "^4.14.1",
```

Comment:
why downgrade?

Reply:
It's not downgrade actually. Changed to the latest on a previous commit then to this one which matches with acala api